pkg/report/output/saas/types/types.go

```diff
@@ -23,11 +23,16 @@ type Meta struct {
 }
 
 type BearerReport struct {
-	Meta       Meta                         `json:"meta" yaml:"meta"`
-	Findings   map[string][]security.Result `json:"findings" yaml:"findings"`
-	DataTypes  []dataflowtypes.Datatype     `json:"data_types" yaml:"data_types"`
-	Components []dataflowtypes.Component    `json:"components" yaml:"components"`
-	Errors     []dataflowtypes.Error        `json:"errors" yaml:"errors"`
-	Files      []string                     `json:"files" yaml:"files"`
+	Meta       Meta                      `json:"meta" yaml:"meta"`
+	Findings   map[string][]SaasFinding  `json:"findings" yaml:"findings"`
+	DataTypes  []dataflowtypes.Datatype  `json:"data_types" yaml:"data_types"`
+	Components []dataflowtypes.Component `json:"components" yaml:"components"`
+	Errors     []dataflowtypes.Error     `json:"errors" yaml:"errors"`
+	Files      []string                  `json:"files" yaml:"files"`
 	// Dependencies []dataflowtypes.Dependency    `json:"dependencies" yaml:"dependencies"`
 }
+
+type SaasFinding struct {
+	Finding        *securitytypes.Finding
+	SomeExtraField string `json:"some_extra_field" yaml:"some_extra_field"`
```

Comment:
To be removed - proof of concept to understand how we would include field in SaaS Findings but not Security Findings